polymetis/polymetis/python/polysim/envs/bullet_manipulator.py

```diff
@@ -84,6 +85,36 @@ def __init__(
         else:
             raise Exception(f"Unknown robot definition extension {ext}!")
 
+        if extract_config_from_rdf:
+            log.info("************ CONFIG INFO ************")
+            num_joints = self.sim.getNumJoints(self.robot_id)
+            for i in range(num_joints):
+                (
+                    jointIdx,
+                    jointName,
+                    jointType,
+                    qIndex,
+                    uIndex,
+                    flags,
+                    jointDamping,
+                    jointFriction,
+                    jointLowerLimit,
+                    jointUpperLimit,
+                    jointMaxForce,
+                    jointMaxVelocity,
+                    linkName,
+                    jointAxis,
+                    parentFramePos,
+                    parentFrameOrn,
+                    parentIdx,
```

Comment:
Most of these fields aren't being used, so it would be a lot more compact to do
```py
                joint_info = self.sim.getJointInfo(self.robot_id, i)
                log.info("Joint {}".format(joint_info[1].decode("utf-8")))  # jointName
                log.info("\tLimit low : {}".format(joint_info[8]))  # jointLowerLimit
                log.info("\tLimit High: {}".format(joint_info[9]))  # jointUpperLimit
                log.info("\tJoint Damping: {}".format(joint_info[6]))  # jointDamping
```
and link to pybullet docs in a comment as a quick reference for future developers changing this part of the code.